### PR TITLE
`tiledb.version.version` Returns Version As Tuple

### PR DESCRIPTION
**Pull request by nguyenv**
- Modified to synchronize with `tiledb.libtiledb.version()`

### CODE REVIEW

**Comment by nguyenv** [comment]
```
(tiledb-3.9) vivian@mangonada:~/TileDB-Py$ python
Python 3.9.7 (default, Sep 16 2021, 13:09:58)
[GCC 7.5.0] :: Anaconda, Inc. on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import tiledb
>>> tiledb.version.version
(0, 11, 2, 'dev5', 'dirty')
>>> tiledb.__version__
'0.11.2.dev5+dirty'
>>>
```

File: HISTORY.md [hunk 1]
```diff
@@ -3,6 +3,9 @@
 ## Impovements
 * Support for Python 3.10 [#808](https://github.com/TileDB-Inc/TileDB-Py/pull/808)
 
+## API Changes
+* Addition of `tiledb.version()` to return verison as a tuple [#801](https://github.com/TileDB-Inc/TileDB-Py/pull/801)
```
Comment (ihnorton) [inline HISTORY.md:7]: small typo: `verIson` -> `versIon`

**Review by ihnorton — APPROVED** [review]
nit, otherwise LGTM. Thanks!

**Comment by ihnorton** [comment]
Please add a test too before merging (there's an existing VersionTest in `test_libtiledb.py`)

**Pull request merged**